### PR TITLE
#149913977 Create profile picture dropdown

### PR DESCRIPTION
**Pull request by andela-cofor**
#### What does this PR do?
- add a drop down list to the profile picture icon. 

#### Description of Task to be completed?
create a drop down feature to display a drop down list when user hovers on profile picture icon on the navigation bar.

#### How should this be manually tested?
After cloning the repository and installing the required dependencies using 
* Run `yarn install`
* Run  `npm run start:dev -s`
* Hover on the profile picture icon

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
 [#149913977 ](https://www.pivotaltracker.com/story/show/149913977) 

#### Screenshots (if appropriate)
screen shot 2017-08-03 at 2 23 05 pm

#### Questions
N/A

### CODE REVIEW

**Comment by Andela-Didacus** [comment]
@andela-kakpobome LGTM

**Comment by Mbarak-Mbigo** [comment]
Looks okay

File: src/common/navbar/NavBar.component.tsx [hunk 1]
```diff
@@ -23,7 +26,7 @@ export class NavBar extends React.Component <any, any> {
             <div className="logo">
               {
                 (image) ? <img src={image}/>
-                : <h1>CALM</h1>
+                  : <h1>CALM</h1>
```
Comment (andela-kakpobome) [inline src/common/navbar/NavBar.component.tsx]: Change this to simply be the logo

File: src/common/navbar/NavBar.component.tsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import * as React from 'react';
 import Avatar from 'react-toolbox/lib/avatar';
 
+import { Link } from 'react-router-dom';
+
+import '../../../assets/css/font.scss';
```
Comment (andela-kakpobome) [inline src/common/navbar/NavBar.component.tsx]: Add the different headers .eg // styles

File: src/common/navbar/NavBar.scss [hunk 3]
```diff
@@ -125,5 +125,65 @@
...
+        font-size: 1.1rem;
+        text-decoration: none;
+      }
+
+      .content-container {
+        height: 10.7rem;
+        border: solid 1px map-get($grey, 200);
+        border-radius: 3px;
+        padding-top: 0.7rem;
+        background-color: map-get($grey, 100);
```
Comment (andela-kakpobome) [inline src/common/navbar/NavBar.scss]: ### Critical

Background should be first

**Comment by andela-cofor** [comment]
@andela-kakpobome done...

**Pull request merged**